tdrs-backend/docker-compose.local.yml

```diff
@@ -29,6 +29,18 @@ services:
       # Copy in the Localstack setup script to configure any buckets needed
       - ../scripts/localstack-setup.sh:/docker-entrypoint-initaws.d/localstack-setup.sh
 
+  elastic:
+    image: elasticsearch:7.17.6
+    environment:
```

Comment:
For this service, do we need this version to match up with the pinned version of 7.13.4?

Reply:
No, the versions of the elastic libraries are major-version-associated, so as long as we're on 7.x.x for everything, we should be set. I just chose the most recently released image for the 7.x version of elastic, all the client libraries are compatible within the major version